test/unittests/UtilityTest.php

```diff
@@ -702,22 +702,22 @@ public function testLookupBatteryWithStage()
      * Tests that getSourcefields will return an empty array 
      * if no parameters are specified
      *
-     * @covers Utility::getSourcefields()
+     * @covers BVL_Feedback_Panel::getSourcefields()
```

Comment:
These tests should be moved to the corresponding class. `test/unittests/UtilityTest.php` is meant to contain tests for methods within `Utility`. 

That said, i don't think we actually have unit tests for the `BVL_Feedback_Panel` class. For now, I think we can delete these tests and make a note to restore them later.

I'll create a new issue for this task and link this PR.

Reply:
Sure! @johnsaigle I would delete this test shortly.